generator/src/main/java/com/scottlogic/deg/generator/generation/StandardFieldValueSourceEvaluator.java

```diff
@@ -56,52 +51,44 @@
         return validSources;
     }
 
+    public MustContainsFieldValueSource getRequiredValueSources(FieldSpec fieldSpec){
```

Comment:
Rename to `getMustContainsValues` or something that is more indicative of the result